docs/MakeOTFUserGuide.md

```diff
@@ -386,38 +373,10 @@ When bit 9 is on, programs that support the CSS2 font specification standards wi
 
 Note that if any of these three new bits is turned on, with the option `–osbOn <n>`, then `makeotf` will require that all three values be explicitly set to be on or off. This is because, if any of new the bits is set on, `makeotf` will by default set the OS/2 version number to 4. When the version number is 4, then both the on and the off setting has a specific meaning for each bit.
 
-## **Synthetic Glyphs**
-
-MakeOTF includes two Multiple Master fonts built-in, one serif and one sans-serif. With these it can synthesize glyphs that match (more or less) the width and weight of the source font. It requires the glyphs zero and O to be present in the font, in order to determine the required weight and width. If the option `–adds` is used, the list of glyphs to generate will be derived from the concatenation of the following three groups of glyphs:
-  1) Euro
-  2) Apple Symbol glyphs. These glyphs were formerly supplied by the Macintosh ATM™ and the Laserwriter® drivers. This is no longer true for OpenType fonts.
-  3) A miscellany of glyphs missing from some of the Adobe Type Library fonts, which were just a few glyphs short of containing the full Adobe PostScript Standard glyph set.
-
-| Synthetic Glyphs |                |               |
-|------------------|----------------|---------------|
-| € Euro           | ∆ Delta        | Ω Omega       |
-| ≈ approxequal    | ^ asciicircum  | ~ asciitilde  |
-| @ at             | \ backslash    | \| bar        |
-| ¦ brokenbar      | ¤ currency     | † dagger      |
-| ‡ daggerdbl      | ° degree       | ÷ divide      |
-| = equal          | ℮ estimated    | ⁄ fraction    |
-| > greater        | ≥ greaterequal | ∞ infinity    |
-| ∫ integral       | < less         | ≤ lessequal   |
-| ℓ litre          | ¬ logicalnot   | ◊ lozenge     |
-| − minus          | × multiply     | ≠ notequal    |
-| № numbersign     | ½ onehalf      | ¼ onequarter  |
-| ¶ paragraph      | ∂ partialdiff  | ‰ perthousand |
-| π pi             | + plus         | ± plusminus   |
-| ∏ product        | " quotedbl     | ' quotesingle |
-| √ radical        | § section      | ∑ summation   |
-| ¾ threequarters  | 0 zero         |               |
-
-The glyphs are synthesized from the MM fonts which MakeOTF has built-in. It will try to match the glyph width of the zero and the dominant stem width of the target font. However, MakeOTF cannot stretch the MM font data to match very thick strokes, very wide glyphs, and it cannot match the design’s stem contrast.
-
 ---
 
 #### Document Version History
 
 Version 1.0 - Initial version  
-Version 1.5 - Revisions for MakeOTF v2.5 - 2016  
+Version 1.5 - Revisions for `makeotf` v2.5 - 2016  
 Version 2.0 - Convert to Markdown, many minor updates and fixes - Josh Hadley, October 2019  
```

Comment:
Seems like this is enough change to at least bump to 2.1, no?

Reply:
indeed :-)